io/net/htx_nic_devices.py

```diff
@@ -263,7 +213,9 @@ def update_host_peer_names(self):
         """
         host_name = process.system_output("hostname", ignore_status=True,
                                           shell=True, sudo=True).decode("utf-8")
-        peer_name = self.run_command("hostname")[-1]
+        cmd = "hostname"
+        output = self.session.cmd(cmd)
```

Comment:
maybe can drop `cmd` as a variable

Reply:
We are using multiple htx commands in various parts of the code. Hence retaining the variable for readability of the code